lib/modules/networkingmodule/networkingmodule.py

```diff
@@ -11,6 +11,8 @@
 import json
 import numpy as np
 
+CHANNEL_ID = "C03QSC88Y4E"
```

Comment:
This same constant is in two places. Potentially we could have a constants.json file with all similar constants to avoid replication and potential errors